pkg/commands/process/settings/rules/javascript/react/google_analytics.yml

```diff
@@ -28,4 +28,7 @@ metadata:
     Coming soon.
     -->
   dsr_id: DSR-1
+  cwe_id:
+    - 200
```

Comment:
I don't think there's any problem in doing this, but as a general point: 201 is a child of 200 (200 is implied by 201) so I'm not sure we need to list both 

(same comment applies to other rules)